### PR TITLE
chore: No signer for AnchorService

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

**Review by dbcfd — APPROVED** [review]

**Comment by stbrody** [comment]
Do we need any corresponding changes to the AuthenticatedEthereumAnchorService?  I believe that class is why the interface has the ceramic setter in the first place.

https://github.com/ceramicnetwork/js-ceramic/blob/eae99ad9112c5a486cbdfe1e12a30efa812b0e76/packages/core/src/anchor/ethereum/ethereum-anchor-service.ts#L145

File: packages/core/src/ceramic.ts [hunk 1]
```diff
@@ -463,7 +463,6 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
         await this._ipfsTopology.start()
       }
 
-      this.anchorService.signer = this._signer
```
Comment (stbrody) [inline packages/core/src/ceramic.ts:466]: actually, will removing this break the ability of Ceramic to authenticate to the CAS?

Comment (dbcfd) [reply packages/core/src/ceramic.ts:466]: Signer is now in the auth class itself, so anchor service no longer needs it https://github.com/ceramicnetwork/js-ceramic/blob/eae99ad9112c5a486cbdfe1e12a30efa812b0e76/packages/core/src/anchor/auth/did-anchor-service-auth.ts#L24

Comment (stbrody) [reply packages/core/src/ceramic.ts:466]: okay awesome.  Do we even need the AuthenticatedEthereumAnchorService class at all anymore then?

Comment (stbrody) [reply packages/core/src/ceramic.ts:466]: oh wait, yes we do, I see now it overrides the sendRequest method.

But we can probably at least remove the Ceramic setter method from it though?

**Pull request merged**